src/components/profile/ProfileAccountAdminCard.tsx

```diff
@@ -73,7 +81,41 @@ const CountDownText: React.VFC<{ email: string; memberId: string }> = ({ email,
   const { authToken } = useAuth()
   const { id: appId } = useApp()
 
+  useEffect(()=>{
+    const lastSendTime = Number(localStorage.getItem('currentTime'))
```

Comment:
after a few discussions, prefter more readable and meaningful naming
ex: lastVerificationSent, previousClickTime ...